stock_scanner/__openerp__.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    stock_scanner module for OpenERP, Allows managing barcode readers with
+#    simple scenarios
+#    Copyright (C) 2011 SYLEAM Info Services (<http://www.syleam.fr/>)
+#              Sylvain Garancher <sylvain.garancher@syleam.fr>
+
+#    This file is a part of stock_scanner
+#
+#    stock_scanner is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    stock_scanner is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
```

Comment:
Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
Add OCA author
Version 8.0.1.0.0
Remove empty keys